ntbs-service/Pages/Notifications/Edit/PatientDetails.cshtml.cs

```diff
@@ -80,6 +81,18 @@ protected override async Task<IActionResult> PrepareAndDisplayPageAsync(bool isB
             PatientDetails = Notification.PatientDetails;
             await SetNotificationProperties(isBeingSubmitted, PatientDetails);
 
+            if (Notification.PatientDetails?.Country?.IsLegacy ?? false)
+            {
+                var countriesList = Countries.ToList();
+                countriesList.Add(new SelectListItem
+                {
+                    Value = Notification.PatientDetails.Country.CountryId.ToString(),
+                    Text = Notification.PatientDetails.Country.Name
+                });
+                countriesList = countriesList.OrderBy(c => c.Text).ToList();
+                Countries = new SelectList(countriesList, "Value", "Text");
+            }
```

Comment:
Could take this out to a new method, wasn't sure if it was worth it but if anyone has strong feelings regarding it let me know!

Reply:
I have preference towards extracting into another method - especially as it's hypothetically quite a rare occurrence (which probably calls into question whether the below discussion has any merit... but putting it out there anyway).

As discussed offline, I think I prefer moving towards using a `List<SelectListItem>` and additionally think I would prefer for the LegacyCountry to be at the top of the dropdown. The latter part would need to be QAed by Nancy/Tehreem - but I think it is a better UX.

@jasiekmiko have you a preference on this?